indico/modules/rb/client/js/common/timeline/TimelineItem.jsx

```diff
@@ -99,6 +99,7 @@ class TimelineItem extends React.Component {
     actions: PropTypes.exact({
       openBookingDetails: PropTypes.func.isRequired,
     }).isRequired,
+    hideRecurringTooltip: PropTypes.bool,
```

Comment:
just wondering, why `hideRecurrence` and `hideRecurringTooltip` and not the same name everywhere?